src/classes/CW_BatchJobProgressPageControllerTest.cls

```diff
@@ -0,0 +1,56 @@
+/**
+* File Name   :    CW_BatchJobProgressPageControllerTest.cls
+* Description :    B-110984
+*
+* Modification Log
+* ============================================================================
+* Ver Date       Author         Modification
+* --- ---------- -------------- --------------------------
+* 0.1 10/04/2014 Klea Kolaric	Initial version
+*
+*
+*/
+@isTest
```

Comment:
Style issue: classes and triggers should use the "UpperCamelCase" notation. Suggestion: rename accordingly.